cocotb_test/simulator.py

```diff
@@ -298,12 +307,21 @@ def get_define_commands(self, defines):
 
         return defines_cmd
 
+    def get_parameter_commands(self, parameters):
```

Comment:
Since we will not support all simulators at the moment would be nice to add this function to the base class and show some warning/error that this is not supported for this simulator?